packages/fiori/src/Timeline.ts

```diff
@@ -338,6 +342,19 @@ class Timeline extends UI5Element {
 	_onkeydown(e: KeyboardEvent) {
 		const target = e.target as ITimelineItem;
 
+		if (isDown(e) || isRight(e)) {
+
+			this._handleDown();
+			e.preventDefault();
+			return;
+		}
+
+		if (isUp(e) || isLeft(e)) {
+			this._handleLodeMoreUp(e);
```

Comment:
IMO would be better to have consistent names: `_handleDown`, `_handleUp`. Is there a specific reason for the name of this function: `_handleLodeMoreUp`